src/extern/libunwind/Makefile.am

```diff
@@ -59,7 +59,7 @@ NEW_LIB = libunwind.a
 
 OBJCOPY = objcopy
 
-SYMBOL_NAMES = mmap munmap
+SYMBOL_NAMES = mmap munmap dl_iterate_phdr
 
 REDEFINE_ARGS = $(foreach name,$(SYMBOL_NAMES),--redefine-sym $(name)=hpcrun_real_$(name))
```

Comment:
I think that this yields a bad name: hpcrun_real_dl_iterate_phdr, which isn't the real function; it is a replacement.

For mmap and munmap, the hpcrun_real code implements the real function.